src/index.css

```diff
@@ -1,13 +1,136 @@
-:root {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+/* Main container of the page's contents */
```

Comment:
CSS looks nearly spot on to the example style, nice! Double check your responsiveness for mobile, in some I checked it does appear that the hearts jump out the bottom of the text box